articles/azure-functions/deployment-zip-push.md

```diff
@@ -25,7 +25,7 @@ To speed up development, you might find it easier to deploy your function app pr
 For more information, see the [.zip deployment reference](https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file).
 
 >[!IMPORTANT]
-> When you use .zip deployment, any files from an existing deployment that aren't found in the .zip file are deleted from your function app.  
+> When you use .zip deployment, any files from an existing deployment that aren't found in the .zip file are deleted from your function app.Any other files and directories found in the site that aren't being overwritten by the deployment, such as those placed there via FTP or created by your app during runtime, will be preserved.
```

Comment:
Let's try to avoid discussing FTP 😉 
```suggestion
> When you use .zip deployment, any files from the previous deployment are either deleted or updated during a subsequent deployment to your function app. Other files and directories in your function app that weren't part of the previous deployment are maintained.
```

Reply:
I like this!